jepsen/src/jepsen/checker.clj

```diff
@@ -63,6 +66,15 @@
          :subdirectory - A directory within this test's store directory where
                          output files should be written. Defaults to nil."))
 
+(defn noop
+  "Creates an empty non-functional checker. 0 arg arity is deprecated."
+  ([]
+   (reify Checker
+     (check [_ _ model _ _])))
+  ([model]
```

Comment:
Since noop doesn't have any concept of a model, we probably don't need an arg here.

Reply:
agreed, fixing. found a slightly cleaner way that satisfies the test case